Testing/functional/tests/cypress/integration/e2e/user/dependents.js

```diff
@@ -310,18 +305,14 @@ describe("dependents", () => {
             interval: 5000,
         });
 
-        // Click download dropdown
+        // Click download dropdown under Forecasts tab
         cy.get(
-            "[data-testid=download-immunization-forecast-report-btn-" +
-                validDependentHdid +
-                "]"
+            `[data-testid=download-immunization-forecast-report-btn-${validDependentHdid}]`
         ).click({ force: true });
 
-        // Click XLSX
+        // Click CSV
         cy.get(
-            "[data-testid=download-immunization-forecast-report-xlsx-btn-" +
-                validDependentHdid +
-                "]"
+            `[data-testid=download-immunization-forecast-report-xlsx-btn-${validDependentHdid}]`
         ).click({ force: true });
```

Comment:
Seems like this is clicking XLSX button, not CSV.  Typo in comment?

Reply:
Fixed